### PR TITLE
Added support for sending from more than one seed.

### PR DESCRIPTION
**Pull request by frohal**
I've added a seed-property for the input-object, so that when you get balances for two or more seeds, the input-objects keeps information on what seed they belong to, and when you create the outgoing transaction, you can include inputs from different seeds, and they will be signed with their correct key.

I've also simplified the signing-algorithm a bit more after the previous one that was wrong.
There is no need to first handle the case of sec-level 1 and then loop only id seclevel2 or more is chosen. 
Code is easier to read/undersstand if you start the loop at once, on sec-index 0 (the j).
Also - I've put in code that will reitereate on the bundle-hash if seclevel greater than 3 is selected - even if this is still verified elsewhere in the code, it could be possible to support higher levels later...  This is waht the extre %3-calulation is for...

### CODE REVIEW

File: jota/src/main/java/jota/IotaAPI.java [hunk 1]
```diff
@@ -455,7 +455,7 @@ public StoreTransactionsResponse broadcastAndStore(final String... trytes) throw
 
                 GetBalancesResponse balancesResponse = getBalances(100, inputsAddresses);
                 String[] balances = balancesResponse.getBalances();
-
+                System.out.println(StringUtils.join(balancesResponse.getBalances()," - "));
```
Comment (th0br0) [inline jota/src/main/java/jota/IotaAPI.java]: please remove

File: jota/src/main/java/jota/IotaAPI.java [hunk 2]
```diff
@@ -602,7 +603,7 @@ public GetBalancesAndFormatResponse getBalanceAndFormat(final List<String> addre
             long balance = Long.parseLong(balances.get(++i));
 
             if (balance > 0) {
-                final Input newEntry = new Input(address, balance, start + i, security);
+                final Input newEntry = new Input(seed, address, balance, start + i, security);
```
Comment (th0br0) [inline jota/src/main/java/jota/IotaAPI.java]: This call doesn't validate that an address belongs to the seed and a seed shouldn't be necessary here.

Two options exist from my point of view:
- Wrap seed in a `java.util.Optional` 
- Create a `InputWithSeed` class that extends Input and adds the seed property.

File: jota/src/main/java/jota/model/Input.java [hunk 3]
```diff
@@ -107,4 +110,22 @@ public int getSecurity() {
     public void setSecurity(int security) {
         this.security = security;
     }
+
+    /**
+     * Get the seed for this input
+     * 
+     * @return
+     */
+	public String getSeed() {
```
Comment (th0br0) [inline jota/src/main/java/jota/model/Input.java]: This shouldn't be indented

File: jota/src/main/java/jota/model/Input.java [hunk 4]
```diff
@@ -9,22 +9,25 @@
  * @author Adrian
  **/
 public class Input {
-
+	
+	private String seed;
```
Comment (th0br0) [inline jota/src/main/java/jota/model/Input.java]: remove Indentation

File: jota/src/main/java/jota/model/Input.java [hunk 5]
```diff
@@ -9,22 +9,25 @@
...
     private String address;
     private long balance;
     private int keyIndex;
     private int security;
 
     /**
      * Initializes a new instance of the Input class.
+     * @param seed TODO
```
Comment (th0br0) [inline jota/src/main/java/jota/model/Input.java]: remove

File: jota/src/main/java/jota/utils/IotaAPIUtils.java [hunk 6]
```diff
@@ -65,52 +67,41 @@ public static String newAddress(String seed, int security, int index, boolean ch
                 // Get the corresponding keyIndex of the address
                 int keyIndex = 0;
                 int keySecurity = 0;
+                String signatureSeed = oseed;
```
Comment (th0br0) [inline jota/src/main/java/jota/utils/IotaAPIUtils.java]: This doesn't support multi-seed inputs even though I believe this was your original intention?

**Comment by frohal** [comment]
Since refactoring-process now actually does most of the work needed to be able to use several input-seeds in one bundle, I pull back this request.

**Pull request closed without merging**